quickstarts/calico/config.yml

```diff
@@ -1,17 +1,17 @@
 id: a4383e84-ad92-4771-8f58-d0a459542be0
-slug: calico-newrelic
-title: Calico-Newrelic
+slug: calico-prometheus
+title: Calico (Prometheus)
```

Comment:
callout for the I/O team here that the title is being changed; will this be a problem?

Reply:
Title changes are totally cool! It's the slug that we don't want to change

Reply:
@aswanson-nr Thanks for review. Reverted slug change.